rust/core/src/crypto.rs

```diff
@@ -937,6 +786,10 @@ macro_rules! impl_hash_type {
             fn serialize<'se, W: std::io::Write>(&self, serializer: &'se mut Serializer<W>, force_canonical: bool) -> cbor_event::Result<&'se mut Serializer<W>> {
                 serializer.write_bytes_sz(&self.data, self.encoding.to_str_len_sz(self.data.len() as u64, force_canonical))
             }
+
+            fn to_bytes(&self) -> Vec<u8> {
```

Comment:
I didn't put this in every place that has a to/from raw bytes yet. There's a couple places that don't have it but they'll need to be added. Question: Should this **replace** to_raw_bytes()? i.e. to_bytes() will be raw bytes when possible (hashes, addresses, etc) and CBOR bytes if not, or should there be all 3 in cases where there is a raw bytes representation? e.g. `to_cbor_bytes()`, `to_bytes()`, `to_raw_bytes()` to be maximally explicit. Should all 3 be exposed to WASM?

Reply:
@SebastienGllmt what do you think?

Reply:
Do we even want 3 in that case? Or just `to_cbor_bytes()` for everything and then additionally `to_raw_bytes()` for the types that have a raw bytes representation? (crypto stuff, addresses)